### ISSUE TITLE
[10.0]Uncaught TypeError: Cannot read property 'controller' of null

### ISSUE THREAD

**Issue by yueqin** [odoo/odoo#14097]
Impacted versions:
[10.0c]
Steps to reproduce:
      Sometimes i click the menu,it's will throw out this error.
Current behavior:
      Uncaught TypeError: Cannot read property 'controller' of null
Expected behavior:
     
Video/Screenshot link (optional):

![typeerror](https://cloud.githubusercontent.com/assets/11083821/19913265/fa1a4708-a0db-11e6-86c3-52ea22e4b003.png)

**Comment by Yenthe666** [comment]
Hi @yueqin,

I've tried this quite some times but without any result.
Could you give a scenario or flow in which this happens rather often? I see your language is in Chinese too, could you try in another language?

Regards,
Yenthe

**Comment by yueqin** [comment]
Hi @Yenthe666 ,
Thanks,I will try again,

Regrads,
yueqin

**Comment by n37r06u3** [comment]
same bug in chinese client

**Comment by ryansturmer** [comment]
I have observed this on the latest - not easily to reliably reproduce.

**Comment by augustosamame** [comment]
+1 Spanish client. Will intermittently give out this error.

**Comment by Yenthe666** [comment]
Guys, I really need more info on how to reproduce it before we can handle it. At this point it looks like translation (errors) that produce this.

**Comment by danimaribeiro** [comment]
It's difficult to reproduce this problem.
But I can say that only happen when you click in the main menu, and right after you click in another menu like "Products" while Odoo still loading the first menu.

If the server is slow it happens more often this, because the time to load.

**Comment by lambone** [comment]
I also experienced same issue. when I click on the "app" menu
![image](https://cloud.githubusercontent.com/assets/4221783/22014344/329e7d96-dcd8-11e6-8b25-2d1227623776.png)

**Comment by yk2kus** [comment]
facing same issue

**Comment by mlaitinen** [comment]
I get this whenever I click menu items very fast. Seems like a race condition to me.

**Comment by ianathompson** [comment]
just happened to me on Odoo 10 clean installation. (Nginx Reverse Proxy)
Installed the CRM app.
Waited until the installation completed and then the wizard bubbles popped up. Clicked on the CRM app from the top menu bar. It opened the CRM page, but then also popped this controller error. Not able to replicate again.

**Comment by danimaribeiro** [comment]
@Yenthe666 I fixed with a simple check if the active_view is not null.
I could not track down why the active_view is null, or if it should be guaranteed that active_view is never null.

**Comment by fnegrini** [comment]
I've got the same error when I've created a brand new database with CRM, Sales and Purchase modules installed. This error raised when I clicked on Products item on Purchase menu when I didn't have any product (empty list). The error didn't happen when I inserted a new product through Sales menu.

**Comment by ajeetyv** [comment]
I facing same issue on menu click-

Uncaught TypeError: Cannot read property 'controller' of null
http://192.168.126.9:8069/web/content/424-e404813/web.assets_backend.js:1625
Traceback:
TypeError: Cannot read property 'controller' of null
    at Class.get_scrollTop (http://192.168.126.9:8069/web/content/424-e404813/web.assets_backend.js:1625:793)
    at Class.<anonymous> (http://192.168.126.9:8069/web/content/424-e404813/web.assets_backend.js:1625:2084)
    at Class.trigger (http://192.168.126.9:8069/web/content/442-b3f1cb0/web.assets_common.js:3098:180)
    at Class.trigger (http://192.168.126.9:8069/web/content/442-b3f1cb0/web.assets_common.js:3102:148)
    at _notify (http://192.168.126.9:8069/web/content/424-e404813/web.assets_backend.js:1807:573)
    at Object.append (http://192.168.126.9:8069/web/content/424-e404813/web.assets_backend.js:1808:94)
    at Array.<anonymous> (http://192.168.126.9:8069/web/content/424-e404813/web.assets_backend.js:1632:11)
    at fire (http://192.168.126.9:8069/web/content/442-b3f1cb0/web.assets_common.js:541:299)
    at Object.fireWith [as resolveWith] (http://192.168.126.9:8069/web/content/442-b3f1cb0/web.assets_common.js:546:198)
    at Array.deferred.(anonymous function) (http://192.168.126.9:8069/web/content/442-b3f1cb0/web.assets_common.js:548:56)

**Comment by danimaribeiro** [comment]
I think this bug was fixed with this commit:

https://github.com/odoo/odoo/commit/7e98e32959231b68985b444afbcde06bf64a5fb7#diff-e458892e06b811defad979d2804a656e

**Comment by batzaya** [comment]
![image](https://user-images.githubusercontent.com/4206565/33252854-89138d48-d37b-11e7-818c-f73699df6ee8.png)
this error open "project.task" gantt view odoo 10 community

**Comment by Yenthe666** [comment]
Hi @batzaya,

Please update your source code to the latest version. If you can still reproduce it on an official runbot then please do create a new bug report.

Regards,
Yenthe

**Comment by pkpraveen052** [comment]
@Yenthe666 
I can able to reproduce the error. I often see this error when clicked on the menus.
![odoo error](https://user-images.githubusercontent.com/40301280/44740275-a1af0700-ab17-11e8-88a4-2507b2aecc47.PNG)

**Comment by pkpraveen052** [comment]
@Yenthe666 
PLease find the snapshot of the runbot error also 
![odoo error2](https://user-images.githubusercontent.com/40301280/44740301-b7243100-ab17-11e8-9784-8cf4f399e602.PNG)

**Comment by pedrobaeza** [comment]
I'm pretty sure it's because not all assets are still loaded (you click when the browser is still loading).

**Comment by pkpraveen052** [comment]
@pedrobaeza 
How could we restrict this not happen (Should not open up the bug when User does perform such actions.)

**Comment by pedrobaeza** [comment]
Well, that's inherent to the nature of the web client. You should wait until everything is loaded first time.

**Comment by mlaitinen** [comment]
Yes, you should wait until everything is loaded. No, this kind of confusing error message is not acceptable. 

No other web app than Odoo does this, so I wouldn't call it natural for a web client.

**Comment by pkpraveen052** [comment]
@Yenthe666 @pedrobaeza 
This is not a good suggestion. You need to fix or try finding the file causing this issue..
PLease do the needful..

**Comment by piemar1** [comment]
The same error from client Odoo12.0
![Zrzut ekranu z 2019-07-31 12-22-02](https://user-images.githubusercontent.com/12630051/62204676-fe511c80-b38d-11e9-8116-cd17779325e8.png)
